README.md

```diff
@@ -9,10 +9,17 @@ Version 2.0 of Leaflet.awesome-markers is tested with:
 
 For bootstrap 2.x & Fontawesome 3.x use Leaflet.awesome-markers v1.0
 
+Version 1.0 of Leaflet.awesome-markers is tested with:
+- Bootstrap 3.3.7
+- Font Awesome 4.7.0
+- Leaflet > 0.7
+
+Use version 2.0 for newer versions of bootstrap and font-awesome!
+
```

Comment:
I have revisited this now, did not see that there is already something written about version 2.0, my bad.
I would suggest that this part is written a bit better all together, like this.

Version 2.0 of Leaflet.awesome-markers is tested and works with the latest available versions:
- Bootstrap 3 (up to 3.3.7)
- Font Awesome 4.0 (up to 4.7.0)
- Ionicons 1.5.2
- Leaflet 0.5 (up to 1.0.3)

Use version 2.0.x for compatibility with the newer versions of bootstrap, font-awesome and leaflet.

Version 1.0 of Leaflet.awesome-markers is tested and works with:
- Bootstrap 2
- Font Awesome 3.0
- Leaflet 0.5

Use version 1.0.x for compatibility with the older versions of bootstrap, font-awesome and leaflet.